test/integration/fn_mount_cmd.go

```diff
@@ -106,7 +106,7 @@ func validateMountCmd(ctx context.Context, t *testing.T, profile string) {
 	}
 
 	start := time.Now()
-	if err := retry.Expo(checkMount, time.Second, 15*time.Second); err != nil {
+	if err := retry.Expo(checkMount, time.Second, Seconds(15)); err != nil {
```

Comment:
Seconds is not a good name for this function. 

How about "ApproximateSeconds" or "SecondsWithMultiplier"?